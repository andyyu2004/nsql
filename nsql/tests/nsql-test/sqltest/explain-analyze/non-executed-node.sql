-- ensure nodes that aren't run still have profiling information
EXPLAIN ANALYZE TIMING OFF
SELECT
    1
WHERE
    1 = 2
