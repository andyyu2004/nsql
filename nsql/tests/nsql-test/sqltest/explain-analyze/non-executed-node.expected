 projection (:push 1 :ret) in=0 out=0           
   filter :push 1 :push 2 :call :ret in=1 out=0 
     dummy scan in=0 out=1                      
                                                